README.md

```diff
@@ -327,8 +327,82 @@ Templates in React are written inside the JavaScript file using the [JSX languag
 
 # Filters
 
-> TODO
-> Angular's pipe character (|) 
+### AngularJS
+
+AngularJS provides filters to transform data for transform of data.<br />
```

Comment:
This sentence sounds strange. Should be `AngularJS provides filters to transform data.`